src/__tests__/functions.test.ts

```diff
@@ -42,6 +42,38 @@ const mockLocations = [
   },
 ];
 
+const mockCollectInputsReturns = [
```

Comment:
Can we use the return types defined in the public docs (index.ts) here? We want to verify that the return value of collectInputs matches the types we defined, since these don't depend on the type they can be theoretically different.